app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments
+  has_many :likes
+
+  def last_five_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] Please add a method here that updates the number of post as mentioned in the requirements. It can look something like this.
```
def update_post_counter
  author.increment!(:posts_counter)
end
```

Reply:
Done! 😄

Reply:
- Well resolved 👍🏽